package.json

```diff
@@ -2,12 +2,12 @@
     "name": "flame-chart-js",
     "version": "2.0.1",
     "description": "",
-    "main": "dist/index.js",
-    "types": "dist/index.d.ts",
+    "main": "dist/src/index.js",
```

Comment:
Why? We don't have src folder in our dist.

Reply:
It is generating a /src/ folder as on this branch - this is fix for that.... 

npm run build:tsc 

And see...

Reply:
But why is it needed?